css/templates/gallery/v-hide.css

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import './bass-override-vars';
+
+@media (--breakpoint-v-xs) {
+  .v-xs-hide { display: none }
```

Comment:
This and other "v-" breakpoints measure device height instead of width. I used them for styling the vertical elements at the left of the design.